core/src/main/java/hudson/model/Job.java

```diff
@@ -135,6 +136,8 @@
 
     private static final Logger LOGGER = Logger.getLogger(Job.class.getName());
 
+    private static /* not final */ int RSS_LIMIT = SystemProperties.getInteger(Job.class.getName() + ".rssChangeLogLimit", 20);
```

Comment:
```suggestion
    private static /* not final */ int DEFAULT_RSS_LIMIT = SystemProperties.getInteger(Job.class.getName() + ".rssChangeLogLimit", 20);
```

Reply:
Why, given it's always the effective limit?